modules/components/pages/outputs/schema_registry.adoc

```diff
@@ -0,0 +1,276 @@
+= schema_registry
+:type: output
+:status: beta
+:categories: ["Integration"]
+
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Publishes schemas to a schema registry.
```

Comment:
@Jeffail. Is there a specific use case for this connector? Can we describe it here?

Also, is this a community or certified connector? I have assumed certified.

Reply:
LGTM